### PR TITLE
feat: TE7-T1022:  Implement custom drawer UI

### PR DESCRIPTION
**Pull request by jigar-s-simform**
Implement custom UI for drawer

### CODE REVIEW

File: android/app/build.gradle
Comment (amitkajwani-simformsolutions) [inline android/app/build.gradle:1]: Why did this gradle file change in this PR as it is not related to drawer UI?

File: app/components/custom-drawer/CustomDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          <Image
+            source={typeof userDetails?.avatar !== 'number' ? {
+              uri: userDetails?.avatar,
+            } : userDetails.avatar}
+            style={customDrawerStyles.profileImgStyles}
+          />
+          <View style={customDrawerStyles.nameEmailContainer}>
+            <Text style={customDrawerStyles.detailNameText}>
+              {userDetails?.first_name +
+                Strings.emptyString +
```
Comment (amitkajwani-simformsolutions) [inline app/components/custom-drawer/CustomDrawer.tsx]: Instead of Strings.emptyString you can use backticks ``

**Review by amitkajwani-simformsolutions — APPROVED** [review]

File: android/app/build.gradle
Comment (jigar-s-simform) [reply android/app/build.gradle:1]: because made release apk from this branch yesterday as it was the latest code and was working fine without any bugs!

Comment (amitkajwani-simformsolutions) [reply android/app/build.gradle:1]: That is fine but this should not belong to this PR. Raise a separate PR for that and remove it from this PR.

Comment (amitkajwani-simformsolutions) [inline android/app/build.gradle:1]: This change should not be in this PR, and I don't think there is any change in this so you can revert it.
And before raising the PR make sure to review it properly.

**Review by amitkajwani-simformsolutions — CHANGES_REQUESTED** [review]

File: android/app/build.gradle
Comment (jigar-s-simform) [reply android/app/build.gradle:1]: there is a change as you mentioned earlier I removed the changes in build.gradle as it was not related to this pr so it is not showing in the changed files !! that was the change

**Review by sagar-simformsolutions — APPROVED** [review]

**Review by amitkajwani-simformsolutions — APPROVED** [review]

**Pull request merged**